### PR TITLE
File serving

### PR DESCRIPTION
**Pull request by iankchristie**
allows to send large file over ble with multiple readcharacteristic requests.

### CODE REVIEW

File: nodejs/FatBeaconPeripheral/webpageCharacteristic.js [hunk 1]
```diff
@@ -11,13 +11,23 @@ var WebpageCharacteristic = function() {
 
   this._value = Buffer.alloc(0);
   this._updateValueCallback = null;
+  this._mtuSize = 251; //5 less than the specified mtu
```
Comment (cco3) [inline nodejs/FatBeaconPeripheral/webpageCharacteristic.js]: space

Comment (cco3) [inline nodejs/FatBeaconPeripheral/webpageCharacteristic.js]: (two before `//` and one after)

**Comment by iankchristie** [comment]
@cco3 fixed changes. I also dismissed the review because it wasn't going away after I made the changes and then recommitted.

**Comment by iankchristie** [comment]
don't know if that was the right move, but it happened.

**Pull request merged**